content/_help/verify-your-identity/phone-number._es.md

```diff
@@ -1,19 +1,16 @@
 ---
-title: "Número de teléfono y plan telefónico a su nombre "
+title: Verifique su número de teléfono
 category: verify-your-identity
-permalink: /es/help/verify-your-identity/phone-number-and-phone-plan-in-your-name/
+permalink: /es/help/verify-your-identity/phone-number/
 order: 5
+redirect_from:
+  - /es/help/verify-your-identity/phone-number-and-phone-plan-in-your-name/
+  - /es/help/verify-your-identity/phone-number/
```

Comment:
Since this is the same as the permalink, I don't think we need to have a redirect for it. It seems like it's not doing anything at the moment, but I'd worry it could create a looping redirect.

```suggestion
```